src/logging/CTiglLogging.cpp

```diff
@@ -94,6 +95,7 @@ CTiglLogging& CTiglLogging::Instance()
 void CTiglLogging::initLogger()
 {
 #ifdef GLOG_FOUND
+    // TODO (bgruber): this is problematic, as the application linking to TIGL may also use glog and initialize it itself
```

Comment:
You are absolutely right. In fact, we don't use glog. Hence, it's implementation is a bit ... untested. Feel free to change it to whatever is might correct.
Alternatively, we can remove glog completely.